### PR TITLE
create thatgirlprogrammer.md

### PR DESCRIPTION
**Pull request by thatgirlprogrammer**
Fixes #issue_number

### Description
Changed tripple ticks to single ticks and removed spaces in-between brackets in thatgirlprogrammer.md

### Raw.Githack preview link
raw.githack link to page(s) changed.

e.g. https://raw.githack.com/UserName/Repository/Branch/Path/To/File.md

### CODE REVIEW

**Comment by thatgirlprogrammer** [comment]
Created thatgirlprogrammer.md
https://raw.githack.com/thatgirlprogrammer/thatgirlprogrammer.github.io/personal_page/#!pages/vi/profiles/thatgirlprogrammer.md

File: pages/vi/profiles/thatgirlprogrammer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Amy Mullins
+#### Atlanta, GA
+
+```print('Hello there')```
```
Comment (dogi) [inline pages/vi/profiles/thatgirlprogrammer.md]: single not triple ticks

File: pages/vi/profiles/thatgirlprogrammer.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+| **hardware**  | **OS** |
+|---------------|----------|
+| Dell Laptop   | Windows 10 |
+| Raspeberry pi | Linux |
+| iPhone | IOS 14 |
+
+Connect with me:
+Github: [github] (https://github.com/thatgirlprogrammer) 
+Linkedin: [linkedin] (https://www.linkedin.com/in/amy-mullins-0a5858111/)
```
Comment (dogi) [inline pages/vi/profiles/thatgirlprogrammer.md]: no spaces between the brackets

**Review by dogi — CHANGES_REQUESTED** [review]

**Comment by dogi** [comment]
@thatgirlprogrammer very awesome profile page ...
... please fix these little errors

**Review by dogi — APPROVED** [review]

**Pull request merged**